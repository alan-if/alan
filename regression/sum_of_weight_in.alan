The l Isa location
End The l.

The balance Isa object At l
  Description
    Say The this. "is"
  If Sum Of weight Directly In left_scale <> Sum Of weight Directly In right_scale Then
    "not"
  End If.
  "in balance."
End The balance.

The left_scale Isa thing At l
  Container Taking object.
End The left_scale.

The right_scale Isa thing At l
  Container Taking object.
End The right_scale.

Add To Every object
  Has weight 1.
End Add.

The small_pebble Isa object In left_scale
  Has weight 1.
End The small_pebble.

The larger_pebble Isa object In right_scale
  Has weight 2.
End The larger_pebble.

Start At l.
