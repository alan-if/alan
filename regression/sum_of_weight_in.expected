########## sum_of_weight_in ##########

        No warnings or errors detected.



L
The balance is not in balance.

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
